### ISSUE TITLE
2.15.2 - Switch-Node switch icon does not show the "ON" state of input when all Dashboards are closed.

### ISSUE THREAD

**Issue by wongnam** [node-red/node-red-dashboard#498]
### What are the steps to reproduce?
- Inject the ON to the Switch-Node when all Dashboard on all web browsers are closed.
- then run web browser to display Node-Red dashboard 
- you will see the switch icon will not change to ON state.

BTW, It will respond well if you open the browser before taking the steps above so that you have to close all dashboard browser to reproduce the issue.
- update 1: my Dashboard is using self-sign https://. 
- update 2: I found that also Button Node has this issue with label displayed by {{msg.topic}}
### What happens?
Switch-Node switch icon does not show the "ON" state of input when all Dashboards are closed.
 
### What do you expect to happen?
It should be response as expected

### Please tell us about your environment:

- [x] Node-RED-Dashboard version: 2.15.2
- [x] Node-RED version:  0.19.5
- [x] node.js version: v8.12.0
- [x] npm version: 6.4.1
- [x] Platform/OS: Raspberry Pi 3B+ / Raspbian Stretch
- [x] Browser: MS Edge, Google Chrome

### Screen shot and Sample flow:
![image](https://user-images.githubusercontent.com/36041520/57903061-7f703b80-7896-11e9-9d10-353fe8dff50f.png)

![image](https://user-images.githubusercontent.com/36041520/57903091-b6dee800-7896-11e9-9fc2-d610ed269364.png)

```
[{"id":"c95be12e.86f1f","type":"ui_switch","z":"dceadd9d.ba53c","name":"","label":"switch","tooltip":"","group":"bd4c84d3.a8f078","order":40,"width":0,"height":0,"passthru":false,"decouple":"true","topic":"","style":"","onvalue":"ON","onvalueType":"str","onicon":"","oncolor":"","offvalue":"OFF","offvalueType":"str","officon":"","offcolor":"","x":370,"y":1450,"wires":[["786537f4.2a0f98"]]},{"id":"48efac7a.d9ac84","type":"inject","z":"dceadd9d.ba53c","name":"","topic":"","payload":"ON","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":130,"y":1430,"wires":[["c95be12e.86f1f"]]},{"id":"786537f4.2a0f98","type":"debug","z":"dceadd9d.ba53c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":610,"y":1450,"wires":[]},{"id":"dadd3f6f.c1955","type":"inject","z":"dceadd9d.ba53c","name":"","topic":"","payload":"OFF","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":130,"y":1480,"wires":[["c95be12e.86f1f"]]},{"id":"bd4c84d3.a8f078","type":"ui_group","z":"","name":"Testing SW/Button","tab":"71576856.c06de8","order":2,"disp":true,"width":"6","collapse":false},{"id":"71576856.c06de8","type":"ui_tab","z":"","name":"Testing","icon":"fa-edit","order":7}]
```

**Comment by wongnam** [comment]
I saw the bug fixed in #497 , i will close this case.
### BTW, I also found that Button-Node also has the same issue like this.

![image](https://user-images.githubusercontent.com/36041520/57905598-45586700-78a1-11e9-80cb-921b7494739d.png)

![image](https://user-images.githubusercontent.com/36041520/57905541-1b9f4000-78a1-11e9-92db-2b02c557ee9d.png)

**Comment by dceejay** [comment]
The label is NOT part of the state of a widget  (like on or off). If you need to set the label when a client connects then use the ui_control node to detect the connection and inject the label you require.

**Comment by hklages** [comment]
... hmmm. But the state of the switch also does not update. It did it the release before.

**Comment by dceejay** [comment]
the switch was fixed in #497 - code is in master - not yet released on npm.

**Comment by wongnam** [comment]
> The label is NOT part of the state of a widget (like on or off). If you need to set the label when a client connects then use the ui_control node to detect the connection and inject the label you require.

I used the label inject for long time ago without any issue. the problem is just happen after update to 2.15.2
- the symptom: the label will not remember in button not after browser close.

**Comment by dceejay** [comment]
the label was never remembered. There is / was no code to do so.

**Comment by dceejay** [comment]
Much clearer on a Monday and a different issue #499.
There was indeed an "enhancement" that improved the ui_template behaviour but broke the switch and button. Now fixed by 7b88d45c552c967b8d5ff61e290a0aa2acb45fb4. Will be in 2.15.3 when released.
